potatobot/potatobot.py

```diff
@@ -116,10 +119,21 @@ def _handle_single_post(self, post):
                              text=post_text,
                              id=post["nr"],
                              status=post_status)
-        for i in self._post_handlers:
-            ret = i(post_info)
-            if ret is not None:
-                self._network.create_followup(post, ret)
+        if self._post_as_answer:
+            final_answer = ""
+            for i in self._post_handlers:
+                ret = i(post_info)
+                if ret is not None:
+                    final_answer = "%s<p></p>%s" % (final_answer, i(post_info))
+            if final_answer and not [d for d in post[
```

Comment:
More idiomatic is `any`, i.e.

```
if final_answer and not any(d for d in post['change_log'] if d['type'] == 'i_answer'):
    .
```

I'm pretty sure all other strings in the codebase use double-quotes, so do that.